br/pkg/restore/client.go

```diff
@@ -2101,15 +2101,16 @@ func (rc *Client) RestoreKVFiles(
 		return errors.Trace(err)
 	})
 
+	if err = eg.Wait(); err != nil {
```

Comment:
why move this here?

Reply:
The log info below need `skipFile`, which is updated by these worker functions. So we need to wait these functions to finish at first, and then print the log Info.